app/models/like.rb

```diff
@@ -1,4 +1,17 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :increment_likes_counter
+  after_destroy :decrement_likes_counter
+
+  private
+
+  def increment_likes_counter
+    post.increment!(:likes_count)
+  end
+
+  def decrement_likes_counter
+    post.decrement!(:likes_count)
+  end
```

Comment:
- [ ] As highlighted by the previous reviewer, the column in the `Post` model is `:likes_counter` and not `:likes_count`. To avoid errors, you need to increment the right column `:likes_counter`. Kindly implement the change. 👍 😎 

#### Hint:
```ruby
post.increment!(:likes_counter)
```